### ISSUE TITLE
[Accessibility] Unable to navigate between action buttons using `Tab` in confirmation dialog

### ISSUE THREAD

**Issue by EastSun5566** [microsoft/vscode#190368]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.81.1
- OS Version: macOS Ventura 13.4.1

Steps to Reproduce:

1. Open a new editor tab with `Cmd` + `N`
2. Type some text
3. Close the tab using `Cmd` + `W`; a confirmation dialog appears, reminding you to save before closing
4. Navigate between the "Save," "Don't Save," and "Cancel" buttons using the `Tab` key

Hi VSCode team,

Thanks for all your hard work. 

I came across this related to accessibility that I couldn't find in the known issues. As mentioned above, the action buttons in the confirmation dialog should be tappable.

<img width="479" alt="截圖 2023-08-14 上午12 46 45" src="https://github.com/microsoft/vscode/assets/26245542/fae2d638-37b1-4b2f-b287-23b70f65e42c">

I'd love to help with this if it's ok. Thanks!

**Comment by meganrogge** [comment]
thanks for the issue . this is working for me - maybe try insider's and/or see what program might be intercepting tab for you
https://github.com/microsoft/vscode/assets/29464607/56917319-4793-4a95-bfd5-dbf2ceeb2986

**Comment by EastSun5566** [comment]
Hi, @meganrogge Thanks for quick response!

So, I've tried a few things – both on the insider and another laptop of mine. As you can see in the video, the tab key just doesn't seem to work only happens when the confirmation dialog pops up. I'm quite confident that no program is messing with the tab key for me. But well, I guess if someone else runs into this then tackle it.

Thanks again!

https://github.com/microsoft/vscode/assets/26245542/876d843a-ddbf-44a0-a7ea-98bbb7374edb
